platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -85,6 +85,7 @@
   @Nullable
   private Bundle savedInstanceState;
   private boolean isStarted;
+  private OnRemoveUnusedStyleImagesListener defaultOnRemoveUnusedStyleImagesListener = null;
```

Comment:
NIT: no need for `null` init

Reply:
we currently lazy initialize `defaultOnRemoveUnusedStyleImagesListener`, since it's created and added for every map initially.  We can initialize the field directly instead.

Reply:
^ discard above if https://github.com/mapbox/mapbox-gl-native/pull/14610#discussion_r282091600 picked up

Reply:
removed `defaultOnRemoveUnusedStyleImagesListener` and moved to core.